BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java

```diff
@@ -0,0 +1,181 @@
+	}
+
+	public WebElement getAges(String value) {
+		return driver.findElement(By.xpath("//div[@about='/products/" + value + "']/div/div/div[3]"));
+	}
+
+	/**
+	 * Function to verify url and banner title
+	 * 
+	 * @param menu          pass menu text
+	 * @param expectedUrl   pass expected URL
+	 * @param expectedTitle pass expected title
+	 */
+	public void verifyUrlAndTitle(String menu, String expectedUrl, String expectedTitle) {
+		Action.explicitWaitForElementTobeclickable(header.getMenuItem(menu), 30);
+		Action.performActionwithExtentInfoLog(header.getMenuItem(menu), "click",
+				"Clicking on : " + header.getMenuItem(menu).getText());
+		Action.verifyPageUrl(expectedUrl);
+		String actualTitle = bannerTitle.getText();
+		Action.printAndAssert(actualTitle, expectedTitle);
+	}
+
+	/**
+	 * Function to verify header and followed links
+	 * 
+	 * @param menu        pass menu text
+	 * @param heading     pass heading
+	 * @param link        pass link
+	 * @param expectedUrl pass expected URL
+	 */
+	public void verifyHeadingsAndLinks(String menu, String heading, String link, String expectedUrl) {
+		Action.explicitWaitForElementTobeclickable(header.getMenuItem(menu), 30);
+		Action.performActionwithExtentInfoLog(header.getMenuItem(menu), "click",
+				"Clicking on : " + header.getMenuItem(menu).getText());
+		Action.explicitWait(getHeadings(heading), 30);
+		boolean eleDisplayed = getHeadings(heading).isDisplayed();
+		if (eleDisplayed) {
+			extentPassLog("Header displayed : ", true);
+			String head = getHeadings(heading).getText();
+			Action.printAndAssert(head, heading);
+			links(link).click();
+			extentInfoLog("Clicked on : ", link);
+			Action.verifyPageUrl(expectedUrl);
+		} else
+			extentFailLog("Header displayed : ", false);
+	}
+
+	/**
+	 * Function to verify article links
+	 * 
+	 * @param menu        pass menu text
+	 * @param headerTitle pass header title
+	 * @param article     pass article
+	 * @param readMore    pass read more link
+	 * @param expectedUrl pass expected URL
+	 */
+	public void verifyArticles(String menu, String headerTitle, String article, String readMore, String expectedUrl) {
+		Action.explicitWaitForElementTobeclickable(header.getMenuItem(menu), 30);
+		Action.performActionwithExtentInfoLog(header.getMenuItem(menu), "click",
+				"Clicking on : " + header.getMenuItem(menu).getText());
+		Action.explicitWait(relatedContentHeader, 30);
+		boolean eleDisplayed = relatedContentHeader.isDisplayed();
+		if (eleDisplayed) {
+			extentPassLog("Related Content header displayed : ", true);
+			String actualHeader = relatedContentHeader.getText();
+			Action.printAndAssert(actualHeader, headerTitle);
+			Action.explicitWait(articleName(article), 30);
+			String actualArticle = articleName(article).getText();
+			Action.printAndAssert(actualArticle, article);
+			Action.explicitWait(readMoreLink(readMore), 30);
+			readMoreLink(readMore).click();
+			extentInfoLog("Clicked on : ", "READ MORE link");
```

Comment:
replace the click and log with "Action.performActionwithExtentInfoLog"
@vaibhavNagvekarSJI

Reply:
fixed

Reply:
Verified @vaibhavNagvekarSJI